---
layout: default
title: "Research"
---

## Publications

[10] **Ma, S.**, Ren, B., Mallick, H., et al. (2021), A statistical model for describing and simulating microbial community profiles. ***PLOS Computational Biology***

[9] Kondo, A., **Ma, S.**, Lee, M.M.Y., et al. (2021), Highly multiplexed image analysis of intestinal tissue sections in patients with inflammatory bowel disease. ***Gastroenterology*** 

[8] Mallick, H., Rahnavard, A., McIver, L.J., **Ma, S.**, et al. (2021), Multivariable Association Discovery in Population-scale Meta-omics Studies. ***PLOS Computational Biology*** 

[7] Dantzler, K., **Ma, S.**, Ngotho, P., et al. (2019), Naturally acquired immunity against immature Plasmodium falciparum gametocytes. ***Science Translational Medicine*** 

[6] **Ma, S.\***, Ogino, S.\*, Parsana, P., et al. (2018), Continuity of transcriptomes among colorectal cancer subtypes, based on meta-analys. ***Genome Biology***

[5] Obaldia, N., Meibalan, E., Sa, J. M., **Ma, S.**, et al. (2018), Bone marrow is a major parasite reservoir in Plasmodium vivax infection. ***MBio***

[4] De Niz, M., Meibalan, E., Mejia, P., **Ma, S.**, et al. (2018), Plasmodium gametocytes display homing and vascular transmigration in the host bone marrow. ***Science advances***

[3] Mallick, H.*, **Ma, S.**, Franzosa, E.A., et al. (2017). Experimental design and quantitative analysis of microbial community multi’omics. ***Genome Biology***

[2] Sinha, R., Abu-Ali, G., Vogtmann, E., Fodor, A.A., Ren, B., Amir, A., Schwager, E., Crabtree, J., **Ma, S.**, et al (2017), Assessment of variation in microbial community amplicon sequencing by the Microbiome Quality Control (MBQC) project consortium, ***Nature Biotechnology***

[1] Pelle, K.G., Oh, K., Buchholz, K., Narasimhan, V., Joice, R., Milner, D.A., Brancucci, N.M.B., **Ma, S.**, et al (2015), Transcriptional profiling defines dynamics of parasite tissue sequestration during malaria infection, ***Genome Medicine***

## Working papers

[4] **Ma, S.**, Li, H., A tensor decomposition model for longitudinal microbiome studies, ***in review***

[3] **Ma, S.**, Shungin, D., Mallick, H., et al., Population structure discovery in meta-analyzed microbial mommunities and inflammatory bowel disease, ***in review***

[2] **Ma, S.**, Li, H., Flexible Ising modeling for co-colonization in longitudinal microbiomes, ***in preparation***

[1] **Ma, S.**, Huttenhower, C., Janson, L., A flexible framework for novel health-microbiome associations and controlling false discoveries, ***in preparation***

## Invited talks

[5] ***A Statistical Model for Simulating and Testing for Microbiomes***, Biostatistics Seminar Series, Department of Biostatistics, Epidemiology, and Informatics, University of Pennsylvania. Philadelphia, PA, 2021

[4] ***Testing Cell-Cell Interactions in Imaging Mass Cytometry Data***, IMC Working Group Meeting, Perelman School of Medicine, University of Pennsylvania. Philadelphia, PA, 2021

[3] ***Meta-Analysis of Population Structure in the IBD Gut Microbiome***, Microbiome Working Group Meeting, Harvard Chan Microbiome in Public Health Center. Boston, MA, 2019

[2] ***Meta-Analysis of Population Heterogeneity in IBD Patients’ Gut Microbiome***, CMIT Work-in-Progress Meeting, Center for Microbiome Informatics and Therapeutics, Massachusetts Institute of Technology. Cambridge, MA, 2017

[1] ***Subtype Continuity Revealed by Meta-analysis of the Colorectal Cancer Transcriptome***, Department of Biostatistics Genomics Meeting, Dana-Farber Cancer Institute. Boston, MA, 2015

## Conference presentations

[5] ***Tensor Decomposition of Longitudinal Microbiomes***, Joint Statistical Meetings. Virtual Conference, 2021

[4] ***Tensor Decomposition of Longitudinal Microbiomes***, ENAR Spring Meeting. Virtual Conference, 2021

[3] ***SparseDOSSA: A Statistical Model for Simulating Realistic Microbial Community Profiles***, Joint Statistical Meetings. Virtual Conference, 2020

[2] ***Population Structure Discovery in Meta-Analyzed Microbial Communities***, Confernece on Intelligent Systems for Molecular Biology. Chicago, Il, 2018

[1] ***Population Structure Discovery in Meta-Analyzed Microbial Communities***, Joint Statistical Meetings. Baltimore, MD, 2017

## Softwares

[2] [SparseDOSSA 2](https://huttenhower.sph.harvard.edu/sparsedossa2/): A statistical model for describing and simulating microbial community profiles

[1] [MMUPHin](https://www.bioconductor.org/packages/release/bioc/html/MMUPHin.html): Meta-analysis Methods with Uniform Pipeline for Heterogeneity in Microbiome Studies
